sectionthree.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	f "fmt"
+	"fibonacci"
+)
+
+func Hitung(i int, j int){
+	f.Println(i)
+	f.Println(j)
+	
+	for index := 2; index < 10; index++{
+		fij := i + j

```

Comment:
mendingan fij initialize di luar for aja, biar tiap looping ga prlu initialize lagi